app/src/common/components/MovieList/MovieElement/MovieElement.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './MovieElement.scss';
+
+import Poster from './Poster/Poster';
+import Genres from './Genres/Genres';
+import Title from './Title/Title';
+import ReleaseYear from './ReleaseYear/ReleaseYear';
+
+export default class MovieList extends React.Component {
```

Comment:
After spreading `{...movie}` this component will also change:
```jsx
const MovieElement = ({ title, release_date, poster_path, genres }) => (
  <div styleName="wrapper">
    <Poster src={poster_path} title={title} />
    <div styleName="row">
      <Title title={title} />
      <ReleaseYear releaseDate={release_date} />
    </div>
    <Genres genres={genres} />
  </div>
);

export default MovieElement;
```

Reply:
done